cli/config.go

```diff
@@ -4,65 +4,211 @@
 package cli
 
 import (
-	"fmt"
-	"log"
+	"io"
 	"os"
+	"reflect"
+	"strconv"
 
 	"github.com/mainflux/mainflux/pkg/errors"
+	mfxsdk "github.com/mainflux/mainflux/pkg/sdk/go"
 	"github.com/pelletier/go-toml"
+	"github.com/spf13/cobra"
 )
 
+type remotes struct {
+	ThingsURL       string `toml:"things_url"`
+	UsersURL        string `toml:"users_url"`
+	ReaderURL       string `toml:"reader_url"`
+	HTTPAdapterURL  string `toml:"http_adapter_url"`
+	BootstrapURL    string `toml:"bootstrap_url"`
+	CertsURL        string `toml:"certs_url"`
+	MsgContentType  string `toml:"msg_content_type"`
+	TLSVerification bool   `toml:"tls_verification"`
+}
+
+type filter struct {
+	Offset          string `toml:"offset"`
+	Limit           string `toml:"limit"`
+	RawOutput       string `toml:"raw_output"`
+	Name            string `toml:"name"`
+	Contact         string `toml:"contact"`
+	Email           string `toml:"email"`
+	Metadata        string `toml:"metadata"`
+	MsgContentType  string `toml:"msg_content_type"`
+	TLSVerification string `toml:"tls_verification"`
```

Comment:
Is `tls_verification` is needed here in filters ?

Reply:
Since I am not using it, let me remove it